quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/DatabaseContainer.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.it.junit5.extension;
+
+import java.time.Duration;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.MariaDBContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class DatabaseContainer {
+
+    static final String DEFAULT_PASSWORD = "Password1!";
+
+    private final String alias;
+    private JdbcDatabaseContainer container;
+
+    DatabaseContainer(String alias) {
+        this.alias = alias;
+    }
+
+    void start() {
+        container = createContainer()
+                .withDatabaseName("keycloak")
+                .withUsername(getUsername())
+                .withPassword(getPassword());
+
+        container.withStartupTimeout(Duration.ofMinutes(5)).start();
+    }
+
+    boolean isRunning() {
+        return container.isRunning();
+    }
+
+    String getJdbcUrl() {
+        return container.getJdbcUrl();
+    }
+
+    String getUsername() {
+        return "keycloak";
```

Comment:
bind it to a `static` field ? e.g. `DEFAULT_USERNAME` ?

Reply:
Why? It should be in the constant pool, doesn't it? Different than the default password, this one is only used within this class so I did not find it necessary to create a constant.